content/commands/ts.add/index.md

```diff
@@ -177,6 +178,30 @@ This override is effective only for this single command and does not set the tim
 This argument has no effect when a new time series is created by this command.
 </details>
 
+<details open><summary><code>IGNORE ignoreMaxTimediff ignoreMaxValDiff</code></summary> 
+
+is the policy for handling insertion of samples that may be duplicates of existing samples according to the following rules:
+
+  - `ignoreMaxTimeDiff`: the system considers a new insertion a duplicate if the time difference does not exceed the maximum allowed. This value defaults to the `IGNORE_MAX_TIME_DIFF` configuration parameter.
+  - `ignoreMaxValDiff`: the system considers a new insertion a duplicate if the value difference does not exceed the maximum allowed. This value defaults to the `IGNORE_MAX_VAL_DIFF` configuration parameter.
+
+  When not specified: set to the global [IGNORE_MAX_TIME_DIFF]({{< baseurl >}}/develop/data-types/timeseries/configuration#ignore_max_time_diff) and [IGNORE_MAX_VAL_DIFF]({{< baseurl >}}/develop/data-types/timeseries/configuration#ignore_max_val_diff), which are, by default, both set to 0.
+
+  These parameters are used when creating a new time series to set the per-key parameters, and are ignored when called with an existing time series (the existing per-key configuration parameters is used).
+
+For each call to `TS.ADD`, if the following condition is met:
+
+```
+if ((series is not a compaction) &&
+    (series' DUPLICATE_POLICY == LAST) &&
+    (timestamp ≥ max_timestamp) &&
+    (timestamp - max_timestamp ≤ ignoreMaxTimeDiff) &&
+    abs(value - value_at_max_timestamp) ≤ ignoreMaxValDiff))
+```
+then ignore this sample.
```

Comment:
We should explain `max_timestamp` (it is the timestamp of the sample with the largest timestamp in the time series). Also `value_at_max_timestamp` (the value at that `max_timestamp`).